multi-node/aws/build

```diff
@@ -15,10 +15,11 @@ if [ -n "$(git diff --shortstat 2> /dev/null | tail -n1)" ]; then
 fi
 
 echo "Building kube-aws ${VERSION}..."
-link_path="${PWD}/gopath/src/github.com/coreos/"
+rm -rf ./gopath
```

Comment:
I just don't want that gopath dir hanging around. I want to discourage people from interacting with the gopath directory in general (erroneously go getting packages in there), so it's cleaned up after the build.